script.js

```diff
@@ -1,62 +1,55 @@
-const imgArr=["./num0.jpg", "./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg", "./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];
-var clickedNumArr=[]; 
-var clickedIdArr=[];
-var randImgArr=[ "./num0.jpg","./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg","./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];
+const imgArr = Array(20).fill().map((_, i) => `./num${i}.png`);
+
+let clickedNumArr=[];
+let clickedIdArr=[];
+
+let randImgArr = imgArr.slice();
 
 // Randomize img locations 
 function shuffle(o) {
-  for(var j, x, i = o.length; i; j = parseInt(Math.random() * i), x = o[--i], o[i] = o[j], o[j] = x);
-  return o;
+  return o.sort(() => (Math.random() > 0.5) ? 1 : -1);
 };
-var random = shuffle(randImgArr);
-var timeOut;
 
+let random = shuffle(randImgArr);
+let timeOut;
 
-//Game play code
+// Game play code
 const images = document.querySelectorAll("#images img");
-    //flip when clicked
+
+    // Flip when clicked
     images.forEach(img => 
-      img.onclick= function () {
+      img.addEventListener('click', function (event) {

```

Comment:
Why is this part indented with 4 spaces? It's not really inside any scope.

Also, event listeners have better support, and allow you to do more things than `onclick`.

Reply:
actually it's not indented, it was for me to say that flipping is the first subpart of the game play. no problem if it's not the standard thing to do for readability and such.

I used an event listener at some point. actually I used multiple options for everything while searching. I'll try it with event listeners to the end this time and see where it goes.